### DISCUSSION TITLE
K8s 1.30 - Formalize docs release strategy with docs localization owners

### DISCUSSION THREAD

**Discussion by drewhagen** [kubernetes/sig-release#2409] [General]
Hello 👋, I'm Drew Hagen, the SIG Release Docs lead for the 1.30 release. I'm opening this discussion to coordinate the docs release (scheduled tentatively for [Wednesday April 17, 2024](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.30#timeline) with localization owners. I look forward to working with everyone! Let me know if you have any questions! Thank you! 🙏🚀

cc: @kubernetes/sig-docs-fr-owners @kubernetes/sig-docs-de-owners @kubernetes/sig-docs-hi-owners @kubernetes/sig-docs-id-owners @kubernetes/sig-docs-ja-owners @kubernetes/sig-docs-ru-owners @kubernetes/sig-docs-pl-owners @kubernetes/sig-docs-ko-owners @kubernetes/sig-docs-pt-owners @kubernetes/sig-docs-es-owners @kubernetes/sig-docs-zh-owners @kubernetes/sig-docs-vi-owners @kubernetes/sig-docs-uk-owners @kubernetes/sig-docs-it-owners

**Comment by seokho-son** [comment]
Hello @drewhagen 
I am one of sig-docs-ko-owners. (as well as one of the localization subproject leads)
Thanks for the notification! :)

**Comment by howieyuen** [comment]
Hi @drewhagen 
I am one of sig-doc-zh-owners, let's rock and roll for kubernetes 1.30 !!!

**Comment by MaxymVlasov** [comment]
Hi there, from sig-docs-uk-owners :)

**Comment by nasa9084** [comment]
Hello, from ja owners, another owner has recently busy, let me know if we need to do something related to ja docs

**Comment by mfilocha** [comment]
sig-docs-pl-owners present!

**Comment by electrocucaracha** [comment]
Hola,

I am one of sig-doc-es-owners

**Comment by perriea** [comment]
Bonjour 🇫🇷 

sig-doc-fr-owners present!

**Comment by shurup** [comment]
I didn't realise each localisation team should reply here, but since everyone seems to enjoy it, let's follow the crowd :-D
Hi from sig-docs-ru-owners! Yet, we have no actual questions regarding v1.30 at the moment.

**Comment by drewhagen** [comment]
Hello localization team leads! I don't think any action is required from you, but I wanted to let you know that we are on track for the 1.30 release on April 17, 2024 and all Kubernetes website branches are up to date (main, dev-1.30). Let me know if I can help with anything! Thanks! :)

**Comment by ydFu** [comment]
thanks :>
